app/models/report_form.rb

```diff
@@ -7,17 +8,47 @@ class ReportForm
     attr_accessor "development_#{n}"
   end
 
+  (1..SMART_OBJECTIVES).each do |n|
+    attr_accessor "smart_what_#{n}"
+    attr_accessor "smart_how_#{n}"
+  end
+
   def development_as_json
     (1..DEVELOPMENT_OBJECTIVES).map { |n| send("development_#{n}") || '' }
   end
 
+  def smart_as_json
+    (1..SMART_OBJECTIVES).map do |n|
+      {
+        what: send("smart_what_#{n}") || '',
+        how: send("smart_how_#{n}") || ''
+      }
+    end
+  end
+
   def self.from_report(report)
```

Comment:
Yeah, this may work. Although the whole idea was that there is 1 `Report` model, which is Active Record. And then a few Active Models which will be used to work with different parts of the Report. In my current story, I'll already be creating a new Form Model. So the Report would have to have multiple factory methods. 

Maybe really the cleanest way is to create a new factory class to manage all those potential forms. I'll take a look when implementing the new feature..